WinUIGallery/ControlPages/MapControlPage.xaml.cs

```diff
@@ -0,0 +1,63 @@
+﻿//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+using System;
+using System.Collections.Generic;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Windows.Devices.Geolocation;
+
+namespace WinUIGallery.ControlPages
+{
+    public sealed partial class MapControlPage : Page
+    {
+        public MapControlPage()
+        {
+            this.InitializeComponent();
+
+            var myLandmarks = new List<MapElement>();
+
+            BasicGeoposition centerPosition = new BasicGeoposition { Latitude = 0, Longitude = 0 };
+            Geopoint centerPoint = new Geopoint(centerPosition);
+
+            map1.Center = centerPoint;
+            map1.ZoomLevel = 1;
+
+            BasicGeoposition position = new BasicGeoposition { Latitude = -30.034647, Longitude = -51.217659 };
+            Geopoint point = new Geopoint(position);
+
+            var icon = new MapIcon 
+            {
+                Location = point,
+            };
+
+            myLandmarks.Add(icon);
+
+            var LandmarksLayer = new MapElementsLayer
+            {
+                MapElements = myLandmarks
+            };
+
+            map1.Layers.Add(LandmarksLayer); 
```

Comment:
Depending on how expensive this, we might want to do this in the OnLoaded event. Otherwise, navigation might "lag" as it waits for the page constructor before navigating to the page.